src/molecule/logger.py

```diff
@@ -74,6 +76,29 @@ def get_logger(name=None) -> logging.Logger:
     return logger
 
 
+def section_logger(func: Callable) -> Callable:
+    """Wrap effective execution of a method."""
+
+    def wrapper(*args, **kwargs):
+        self = args[0]
+        get_logger().info(
+            "[info]Running [scenario]%s[/] > [action]%s[/][/]",
+            self._config.scenario.name,
+            underscore(self.__class__.__name__),
+            extra={"markup": True},
+        )
+        rt = func(*args, **kwargs)
+        # section close code goes here
+        return rt
+
+    return wrapper
+
+
+def get_section_loggers() -> Iterable[Callable]:
+    """Return a list of section wrappers to be added."""
+    return [section_logger]
```

Comment:
are you imagining a separate wrapper function for each of the output types?
```python
return [github_actions_grouper, gitlab_sectioner, travis_ci_folder, section_logger]
```
Then the environment var checks would happen during each of those?
or something more like what I had before:
```python
return [ci_section_wrapper, section_wrapper]
```

Reply:
Add them to the logger module. I was considering console initially but due to some circular import issues I decided to go for logger. logger can import console but console does not, this being the main reason.

Yep, I did fancy adding these to enrich but I am afraid that it may not be so easy. If I am correct, there wrappers may require some parameters like section name.

For the moment just extend get_section_loggers to make it return additional wrappens when needed, add them inside the same file and also add `lru_cache()` wrapper on it for performance reasons, as we do not want it to run each time it wraps a class.